homeassistant/components/todoist/calendar.py

```diff
@@ -219,7 +220,7 @@ def handle_new_task(call: ServiceCall) -> None:
                 due_date = datetime(due.year, due.month, due.day)
             # Format it in the manner Todoist expects
             due_date = dt.as_utc(due_date)
-            date_format = "%Y-%m-%dT%H:%M%S"
+            date_format = "%Y-%m-%dT%H:%M:%S"
```

Comment:
This was an unrelated bug, that caused an API when POST-ed to the todoist api due to the missing `:`.